lang/cs/Org.Apache.REEF.Common/Telemetry/IntegerGauge.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Org.Apache.REEF.Utilities.Attributes;
+
+namespace Org.Apache.REEF.Common.Telemetry
+{
+    class IntegerGauge : IMetric<int>
```

Comment:
Compare this class with your DoubleGauge class. Do you see any similarities? Try not to repeat yourself.

Reply:
Abstracted MetricBase class to avoid code repetition